api/deps.edn

```diff
@@ -4,7 +4,9 @@
   "central" {:url "https://repo1.maven.org/maven2/"}
   "clojars" {:url "https://clojars.org/repo"}}
 
- :paths ["src", "resources"]
+ ; clojure makes me want to self harm. If you're feeling suicidal, you'll
+ ; re-arrange the classpath.
+ :paths ["../derived/api/resources" "resources" "src"]
```

Comment:
```suggestion
 ; derived/api/resources must come first so the environments.clj there is found before
 ; the placeholder in src
 :paths ["../derived/api/resources" "resources" "src"]
```
I don't have a problem with joking about how hilarious this fix is but I think we should include some mention of the actual reason for this.